views/007_events.sql

```diff
@@ -123,6 +125,26 @@ AFTER UPDATE ON checks
 FOR EACH ROW
 EXECUTE PROCEDURE insert_check_updates_in_event_queue ();
 
+-- Insert component status updates in event_queue
+CREATE OR REPLACE FUNCTION insert_component_status_updates_in_event_queue() RETURNS TRIGGER AS $$
+DECLARE event_name TEXT;
+BEGIN
+    IF OLD.status = NEW.status THEN
+      RETURN NULL;
+    END IF;
+
+    event_name := 'component.' || NEW.status || '.sync';
```

Comment:
@moshloop Unlike checks, components have a variety of statuses so the event name is created this way.

```
const (
	ComponentStatusHealthy   ComponentStatus = "healthy"
	ComponentStatusUnhealthy ComponentStatus = "unhealthy"
	ComponentStatusWarning   ComponentStatus = "warning"
	ComponentStatusError     ComponentStatus = "error"
	ComponentStatusInfo      ComponentStatus = "info"
)
```